walkthroughs/howto-k8s-cross-cluster/deploy.sh

```diff
@@ -64,33 +117,39 @@ deploy_cloudmap_ns() {
 
         aws servicediscovery create-private-dns-namespace \
             --name "${CLOUDMAP_NAMESPACE}" \
-            --vpc "${VPC_ID}"
+            --vpc "${VPC_ID}" >/dev/null
         echo "Created private-dns-namespace ${CLOUDMAP_NAMESPACE}"
         sleep 5
     fi
 }
 
 deploy_apps_and_mesh() {
     
-    EXAMPLES_OUT_DIR="${DIR}/_output/"
+    EXAMPLES_OUT_DIR="${DIR}/_output"
     mkdir -p ${EXAMPLES_OUT_DIR}
-    
+
     eval "cat <<EOF
-    $(<${DIR}/cluster2.yaml.template)
-    EOF
-    " >${EXAMPLES_OUT_DIR}/cluster2.yaml
+$(<${DIR}/v1beta2/cluster2.yaml.template)
+EOF
+" >${EXAMPLES_OUT_DIR}/cluster2_manifest.yaml
 
     eval "cat <<EOF
-    $(<${DIR}/cluster1.yaml.template)
-    EOF
-    " >${EXAMPLES_OUT_DIR}/cluster1.yaml
+$(<${DIR}/v1beta2/cluster1.yaml.template)
+EOF
+" >${EXAMPLES_OUT_DIR}/cluster1_dummy_arn.yaml
 
-    KUBECONFIG="$HOME/.kube/eksctl/clusters/${CLUSTER1}" kubectl apply -f ${EXAMPLES_OUT_DIR}/cluster1.yaml
-    KUBECONFIG="$HOME/.kube/eksctl/clusters/${CLUSTER2}" kubectl apply -f ${EXAMPLES_OUT_DIR}/cluster2.yaml
+    KUBECONFIG="$HOME/.kube/eksctl/clusters/${CLUSTER1}" kubectl apply -f ${EXAMPLES_OUT_DIR}/cluster2_manifest.yaml
 
+    sleep 120
```

Comment:
Sleeping to wait for resource readiness/creating?

nite: maybe check in a loop with timeout?